nx.json

```diff
@@ -1,22 +1,85 @@
 {
-	"tasksRunnerOptions": {
-		"default": {
-			"runner": "nx/tasks-runners/default",
-			"options": {
-				"cacheableOperations": ["build", "test", "lint", "format"],
-				"accessToken": "ODlkYjAwMjYtMGMyMS00YTI3LWIzNTAtN2RhZjgwYzAwZGJifHJlYWQtd3JpdGU="
-			}
-		}
-	},
+	"$schema": "./node_modules/nx/schemas/nx-schema.json",
 	"targetDefaults": {
 		"build": {
-			"dependsOn": ["^build"]
+			"dependsOn": [
+				"^build"
+			],
+			"inputs": [
+				"default",
+				"^default",
+				{
+					"env": "ALGOLIA_ADMIN"
+				},
+				{
+					"env": "ALGOLIA_APPLICATION"
+				},
+				{
+					"env": "JWE_SECRET"
+				},
+				{
+					"env": "NODE_VERSION"
+				},
+				{
+					"env": "PUBLIC_GIT_PROXY_BASE_URL"
+				},
+				{
+					"env": "PUBLIC_LIX_GITHUB_APP_CLIENT_ID"
+				},
+				{
+					"env": "LIX_GITHUB_APP_CLIENT_SECRET"
+				},
```

Comment:
There is no better way to invalidate the cache? The hardcoded env variables will eventually run out of sync and cause regression bugs that are hard to debug

Reply:
Actually, I did just have a better Idea. 
We could use a runtime input & use the output of `doppler secrets` as an input for the hash. Will try that tomorrow.
https://nx.dev/recipes/running-tasks/customizing-inputs#include-runtime-information-in-the-cache-hash

Reply:
Let's try that!